README.md

```diff
@@ -4,6 +4,7 @@
 
 ![project image](snapshots/Snapshot_6.png)
 ![project image](snapshots/Snapshot_5.gif)
+![project image](snapshots/Snapshot_7.png)
```

Comment:
Would you mind turning this into a gif showing the player entering and leaving the cabin?